js/index.js

```diff
@@ -1 +1,133 @@
 // Your code goes here
+//  mouseover
+//  keydown
+//  wheel
+//  drag / drop
+//  load
+//  focus
+//  resize
+//  scroll
+//  select
+//  dblclick
+
+
+
+//1
+//mouseover
+let nava= document.querySelector(".nav");
+nava.addEventListener("mouseover", function(event) {   
+    event.target.style.color = "blue";
+});
+nava.addEventListener("click",function(e){
+    event.preventDefault();
+})
+
+
+//2
+//click
+
+
+let logo=document.querySelector("h1");
+logo.addEventListener("click",function(e){
+    e.target.textContent="Vrrroooommm!";
+    e.stopPropagation();
+});
+
+
+let section1=document.querySelector(".content-section h2");
+section1.addEventListener("click",function(e){
+    e.target.textContent="HEllO!!";
+    e.stopPropagation();
+})
+
+let pic1=document.querySelector(".content-section img");
+pic1.addEventListener("click",function(e){
+    e.target.src="img/fun.jpg";
+    e.stopPropagation();
+});
+
+
+//3
+//bldclick
+let pic2=document.querySelector(".inverse-content img")
+pic2.addEventListener("dblclick",function(e){
+    e.target.src="img/adventure.jpg";
+    
+});
+
+
+ //4
+let forms=document.querySelector('input[type="text"]');
+forms.style.marginBottom="20px";
+forms.addEventListener("focus",(event)=>{
+    event.target.style.background="yellow";
+});
+
+
+//5
+window.addEventListener("keyup",keyPressUp,false);
+function keyPressUp(key){
+    if(key.keyCode=="69"){//e
+        alert("jump back on the fun bus");
+    }
```

Comment:
For clarity, you might want to add a comment here indicating that you are looking for an `E` key press.